### DISCUSSION TITLE
Add a countdown to standard Osu when using a touch device.

### DISCUSSION THREAD

**Discussion by Hissymaster** [ppy/osu#27871] [Ideas] [answered]
The countdown can also be useful on touchscreen devices on standard Osu. 
Currently, you are expected to immediately react and move your hand over to the next circle. This change would allow touch players to prepeare before the game resumes, similarly to how mouse and tablet users hover over the next circle to continue.
You could have it do so whenever the touch device mod is on.

**Comment by Walavouchey** [comment]
i fail to see how the osu! mode is different on desktop compared to a touch screen regarding this. in lieu of a countdown, you press the orange cursor - that's your time to prepare.

other modes don't have that kind of mechanic and use a countdown instead

**Comment by Hissymaster** [comment]
On touch it just continues without warning...

**Reply by Walavouchey** [reply]
upon testing that seems to indeed be the case...

@ppy/team-client 

unpausing after clicking the on-screen pause button (not just on mobile) has neither a delay nor orange resume cursor, which feels weird

https://github.com/ppy/osu/assets/36758269/5fca0d0f-3b7b-4916-8cc5-fcd011a8dd76

**Reply by peppy** [reply]
If we can make use of the new delayed resume overlay for this case it would be pretty cool.

**Reply by bdach** [reply, marked as answer]
This is actually a regression from https://github.com/ppy/osu/commit/bce3bd55e5a863e52f41598c306a248a79638843.
